### PR TITLE
[Android Auto] Bump car library to 1.2.0

### PR DESCRIPTION
**Pull request by kmadsen**
### Summary of changes

The `1.2.0` Jetpack Car Library is out with many bug fixes, and there is a final version out. Let's bump and release https://developer.android.com/jetpack/androidx/releases/car-app#1.2.0

It is also possible to keep 1.1.0 and upgrade to 1.2.0 downstream, but we have not found a reason to continue to support 1.1.0 because it has issues that were addressed in 1.2.0.

## Pull request checklist:
 - [x] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Run `make update-api` to update generated api files, if there's public API changes, otherwise the `verify-api-*` CI steps might fail.
 - [x] Update [CHANGELOG.md](../CHANGELOG.md) or use the label 'skip changelog', otherwise `check changelog` CI step will fail.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.[version]` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

**Review by pengdev — APPROVED** [review]

File: extension-androidauto/CHANGELOG.md
Comment (alexander-kulikovskii) [inline extension-androidauto/CHANGELOG.md:11]:
```suggestion
* Make the SDK compatible with Jetpack Car Library `v1.2.+`. ([#1828](https://github.com/mapbox/mapbox-maps-android/pull/1828))
```

**Comment by kmadsen** [comment]
Am integrating the new versions and the deprecated bitmap widget constructors compile but change the alignment behavior https://github.com/mapbox/mapbox-navigation-android/pull/6588#issuecomment-1312069197

I think it's all good considering it's experimental, but letting you know

**Comment by kmadsen** [comment]
@pengdev I have been trying to find a work around for `10.10` BitmapWidget with an implementation that also works for `10.9`, but it doesn't work.

We really need `10.10` to not break `10.9`, can you look into it?

Screen Shot 2022-11-11 at 2 18 40 PM

**Review by yunikkk — APPROVED** [review]

**Comment by kmadsen** [comment]
Considering that we want to create a new release after this merges, I'm holding off on merging because we may want to have this in the release https://github.com/mapbox/mapbox-maps-android/pull/1834

**Pull request merged**
